Resources/Prototypes/Entities/Structures/Walls/walls.yml

```
@@ -712,4 +712,31 @@
       - FullTileMask
       layer:
       - GlassLayer
-  - type: Airtight
\ No newline at end of file
+  - type: Airtight
+
+- type: entity
+  id: WallForce
+  name: Force Wall
+  components:
+    - type: ForceWallSpell
+    - type: Tag
+      tags:
+        - Wall
+    - type: Physics
+      bodyType: Static
+    - type: Fixtures
+      fixtures:
+        - shape:
+            !type:PhysShapeAabb
+            bounds: "-0.5,-0.5,0.5,0.5"
+          mask:
+            - FullTileMask
+          layer:
+            - GlassLayer
```

```suggestion
            - WallLayer
```
GlassLayer doesn't block lasers, so I suggest changing it to WallLayer.